src/webhooks/__tests__/registry.test.ts

```diff
@@ -365,277 +376,129 @@ describe('shopify.webhooks.process', () => {
   it('handles the request when topic is already registered', async () => {
     shopify.webhooks.addHandler({
       topic: 'PRODUCTS',
-      registryEntry: {
-        path: '/webhooks',
-        webhookHandler: genericWebhookHandler,
-      },
+      ...genericRegistryEntry,
     });
 
-    const app = express();
-    app.use(parseRawBody);
-    app.post('/webhooks', async (req, res) => {
-      let errorThrown = false;
-      let response: NormalizedResponse = {
-        statusCode: 200,
-        statusText: 'OK',
-      };
-      try {
-        response = await shopify.webhooks.process({
-          rawBody: (req as any).rawBody,
-          rawRequest: req,
-          rawResponse: res,
-        });
-        res.writeHead(response.statusCode);
-        res.end();
-      } catch (error) {
-        errorThrown = true;
-        res.writeHead(error.response.code);
-        res.end();
-      }
-      expect(errorThrown).toBeFalsy();
-      expect(response.statusCode).toBe(200);
-    });
-
-    await request(app)
+    const response = await request(app)
```

Comment:
Loving how this cleaned up this test, didn't realize we had so many instances of the test express app.